client/src/app/site/pages/meetings/modules/meetings-component-collector/projector-button/components/projector-button/projector-button.component.ts

```diff
@@ -22,15 +27,15 @@ export class ProjectorButtonComponent implements OnInit, OnDestroy {
      * The object to project.
      */
 
-    private _object: ProjectionBuildDescriptor | Projectable | null = null;
+    private _object: ProjectionBuildDescriptor | Projectable | ProjectionDialogConfig | null = null;
```

Comment:
I would prefer extending the `SlideOptions` of the `ProjectionBuildDescriptor` instead of adding a new type here.

Reply:
Did that, but a bit differently than you suggested